openlibrary/plugins/openlibrary/stats.py

```diff
@@ -247,21 +239,25 @@ def page_unload_hook():
     graphite_stats.increment(web.ctx.graphiteRequestStats.to_metric())
 
 
-def increment_error_count(key):
+def increment_error_count(key: str) -> None:
     """
     :param str key: e.g. ol.exceptions or el.internal-errors-segmented
     """
     top_url_path = 'none'
     page_class = 'none'
     if web.ctx and hasattr(web.ctx, 'path') and web.ctx.path:
         top_url_path = _get_top_level_path_for_metric(web.ctx.path)
-        page_class = _get_path_page_name(web.ctx.path)
+        page_class = _get_path_page_name()
```

Comment:
Why this change?

Reply:
`_get_path_page_name()` doesn't access any parameters. I went ahead and cleaned this up for all callers (I had missed one originally):

```
> git grep _get_path_name
openlibrary/plugins/openlibrary/stats.py:def _get_path_page_name() -> str:
openlibrary/plugins/openlibrary/stats.py:            self.path_page_name = _get_path_page_name()
openlibrary/plugins/openlibrary/stats.py:        page_class = _get_path_page_name()
```